### DISCUSSION TITLE
2024年9月24日支付宝续费了, 请安排延长有效期

### DISCUSSION THREAD

**Discussion by enbonled** [getlantern/lantern-forum-cn#887] [General]
![付款凭证446333250@qq com](https://github.com/user-attachments/assets/1ac0079d-d37f-4abd-92cc-3b5dcaa0932f)
账号446333250@qq.com

**Comment by MaxMKMax** [comment]
@enbonled 好的. 我帮您安排一下.

**Reply by enbonled** [reply]
Mike, 现在变成了免费版了, 之前这个账号要到2025年6月才到期的, 怎么变成了免费版本.
通过邮箱446333250@qq.com授权登录, 但是将近一天
![03](https://github.com/user-attachments/assets/b0c20fa4-d3b1-4f53-9798-db6d140de3ee)
一直没有收到验证码? 请问怎么解决. 如果是国内的账号原因, 那就把446333250@qq.com这个账号 换成 enbonled@gmail.com把

**Reply by enbonled** [reply]
> @enbonled 好的. 我帮您安排一下.

Mike 麻烦处理一下

**Reply by enbonled** [reply]
> @enbonled 好的. 我帮您安排一下.

Mike依旧没有处理好,  没有收到授权码.
